src/Screen/HistoryScreen/HistoryScreen.jsx

```diff
@@ -16,8 +16,8 @@ const HistoryScreen = () => {
 
 
   return (
-    <div className="flex-row">
-      <SideBar />
+    <div>
+      <SideBar pageWrapId={'page-wrap'} outerContainerId={'outer-container'} />
```

Comment:
Correct me if I am wrong but You are not using these props in sidebar component so why passing it here?

Reply:
These Props are from ```react-burger-menu``` package. To get Wrap the side bar - So, what pageWrapId is doing here is: ```pageWrapID``` is wrapping the rest of the content on page.